lib/code_teams/plugin.rb

```diff
@@ -61,6 +61,16 @@ def self.register_team(team)
       T.unsafe(registry_for_team[self])
     end
 
+    sig { void }
+    def self.bust_caches!
+      all_plugins.each { _1.clear_team_registry! }
```

Comment:
Haven't seen this syntax before! I'm used to `each(&:clear_team_registry!)`

Reply:
ha, yeah not sure why I used that syntax here 😵 Changed it to the more terse `&:` version.